src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Utils/ParserStateResult.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Microsoft.Sbom.Contracts.Enums;
+
+namespace Microsoft.Sbom.Utils;
+
+#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
```

Comment:
I think I'd rather change the EditorConfig preferences, but we can do that in a separate PR, no need to block on this.